packages/svgs/src/scripts/constants.ts

```diff
@@ -0,0 +1,13 @@
+const ICONS_SOURCE_DIR = "src/icons";
```

Comment:
export const ICONS_SOURCE_DIR = "src/icons";

export everything like this. 

the export at the end is really a module.exports pattern